taxi/forms.py

```diff
@@ -17,6 +17,19 @@ class Meta:
         fields = "__all__"
 
 
+class CarSearchForm(forms.Form):
+    model = forms.CharField(
+        max_length=255,
+        required=False,
+        label="",
+        widget=forms.TextInput(
+            attrs={
+                "placeholder": "Search by model"
+            }
+        )
+    )
+
+
 class DriverCreationForm(UserCreationForm):
     class Meta(UserCreationForm.Meta):
         model = Driver
```

Comment:
Verify that the `Driver` model has a `license_number` field, as it is being used in the `DriverCreationForm` and `DriverLicenseUpdateForm`.